### ISSUE TITLE
Duplicate header key error

### ISSUE THREAD

**Issue by alexcucc** [kptfh/feign-reactive#25]
https://github.com/kptfh/feign-reactive/blob/ae31dddc5fef2468fee853ef07abadbb9feff2e3/feign-reactor-jetty/src/main/java/reactivefeign/jetty/client/JettyReactiveHttpResponse.java#L55

When the response contains multiple headers with the same key it throws a generic error, whereas their value should be collapsed together.

**Comment by kptfh** [comment]
Please check if it works in main repository. I mean Playtika supported one

**Comment by alexcucc** [comment]
Yes, the same happens with the Playtika repo, gonna create the same issue there
